frontend/src/components/borrow/MyLoans.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="my-loans-spinner" v-if="isLoading">
+    <spinner />
+  </div>
+  <div v-else class="my-loans">
+   <div class="ml-summary">
+     <tiles-summary
+        label="CURRENTLY BORROWED"
+        amount="145.00"
+        currency="ETH" />
+      <tiles-summary
+        label="MINIMUM MONTHLY RATES"
+        amount="20.00"
+        currency="ETH" />
+    </div>
+    <div class="ml-loans-container">
+      <loan-tile v-for="loan in loans" :key="loan.shortId" :loan="loan" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import Summary from '../common/TilesSummary.vue';
+import Spinner from '../common/Spinner.vue';
+import MyLoanTile from './MyLoanTile.vue';
+import API, {LoanState} from '../../api';
+
+const Component = Vue.extend({
+  name: 'MyLoans',
+  components: {
+    'tiles-summary': Summary,
+    'loan-tile': MyLoanTile,
+    'spinner': Spinner,
+  },
+  created() {
+    setTimeout(async () => {
+      this.isLoading = true;
+      let currentUser = await API.instance().currentUser();
+      let loans = await API.instance().loansByOwner(currentUser);
+      await Promise.all(loans.map(loan => loan.updateStateFromBlockchain()));
+      let _loans = loans.map(({
+          description,
+          parameters: {interestPermil},
+          blockchainState : {loanState}
+        }) => ({ description, interestPermil, loanState }));
+
+      const humanizeAmountsWanted = await Promise.all(
```

Comment:
Can I have some explicit types here?

Reply:
Same for humanizeAmountsGathere and loanTokenSymbols. I think it'd be easier to read that way.